src/main/webapp/assets/modules/technical-detail/technical-detail.con.js

```diff
@@ -25,13 +25,21 @@ techlooper.controller("technicalDetailController", function (utils, connectionFa
     return sprintf("http://vietnamworks.com/%s-%s+at-it-software-i35-en", termView.label, company.name);
   }
 
-  connectionFactory.termStatisticInOneYear($scope.termRequest)
-    .success(function (data, status, headers, config) {
-      $scope.termRequest.skills = data.skills.map(function (skill) {return skill.skillName;});
-      $scope.termStatistic = termService.toViewTerm(data);
-      technicalDetailService.trendSkills($scope.termStatistic);
-      utils.sendNotification(jsonValue.notifications.loaded);
-    })
-    .error(function (data, status, headers, config) {
-    });
+  $scope.loadData = function () {
+    utils.sendNotification(jsonValue.notifications.switchScope, $scope);
+    connectionFactory.termStatisticInOneYear($scope.termRequest)
+      .success(function (data, status, headers, config) {
+        $scope.termRequest.skills = data.skills.map(function (skill) {return skill.skillName;});
+        $.each(data.skills, function (i, skill) {skill.id = i;});
+        $scope.termStatistic = termService.toViewTerm(data);
+        if (!technicalDetailService.trendSkills($scope.termStatistic)) {
+          console.log("Empty Chart");
+        }
+        utils.sendNotification(jsonValue.notifications.loaded);
+      })
+      .error(function (data, status, headers, config) {
```

Comment:
'config' is defined but never used.<br>'headers' is defined but never used.<br>'status' is defined but never used.<br>'data' is defined but never used.